app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -76,27 +75,15 @@ def container_class(application_choice)
       end
     end
 
+    def title_for(application_choice)
+      "<span class=\"app-course-choice__provider-name\">#{application_choice.offered_course.provider.name}</span>
+      <span class=\"app-course-choice__course-name\">#{application_choice.offered_course.name_and_code}</span>".html_safe
```

Comment:
Maybe:

```
      tag.span(application_choice.offered_course.provider.name, class: 'app-course-choice__provider-name') +
      tag.span(application_choice.offered_course.name_and_code, class: 'app-course-choice__course-name')
```

Reply:
I think this code is removed from master now. So rebasing has got rid of it.